components/Blog.js

```diff
@@ -0,0 +1,16 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+export default function News(props) {
+	const { image, imageAlt, name, date, share, title, link } = props;
+	return (
+		<div className='index--blog'>
+			<Image src={image} alt={imageAlt} height='300px' width='200px' />
+			<h2>{name}</h2>
+			<p>{date}</p>
+			<Link href={share}>Share</Link>
```

Comment:
I think there's more to this than just an 'a' tag and a url. Do a quick google on sharing to social media.

Reply:
the buttons are there but not working atm